examples/cluster/plot_cluster_comparison.py

```diff
@@ -116,8 +116,6 @@
         n_clusters=params['n_clusters'], eigen_solver='arpack',
         affinity="nearest_neighbors")
     dbscan = cluster.DBSCAN(eps=params['eps'])
-    optics = cluster.OPTICS(min_samples=30, maxima_ratio=.8,
-                            rejection_ratio=.4)
```

Comment:
Why is optics being removed from this example?

I find this example very useful to have a big picture view of clustering in scikit-learn.

Reply:
The example will be added when we add the `xi` method (the method from the original paper) in #12077. This moves the OPTICS to an intermediate stage for us to focus on the other PR, and then better examples, and putting OPTICS back in examples.